src/language/walker/programAwareRuleWalker.ts

```diff
@@ -20,8 +20,8 @@ import {IOptions} from "../../lint";
 import {RuleWalker} from "./ruleWalker";
 
 export class ProgramAwareRuleWalker extends RuleWalker {
-    protected program: ts.Program;
-    protected typeChecker: ts.TypeChecker;
+    private program: ts.Program;
```

Comment:
Let's go ahead and use parameter properties where possible 👍 
In fact, just merged https://github.com/palantir/tslint/pull/1373 which does this in a few locations